ansible_collections/arista/avd/molecule/eos_designs-l2ls/documentation/devices/BGP-LEAF1.md

```diff
@@ -93,10 +97,12 @@ vlan 100
 
 | Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | Channel-Group |
```

Comment:
Question for reviewers: Do we want to add phone_vlan and associated tagged/untagged mode here?

Reply:
Maybe create a separate table only for phone interfaces with this info in?